setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="reasoner-pydantic",
-    version="1.2.0.4",
+    version="1.3.0.1",
```

Comment:
I believe we should consider going to 2.0 here. Given the change to results.add

Reply:
I agree